files/KoreBuild/scripts/get-dotnet.sh

```diff
@@ -1,101 +0,0 @@
-#!/usr/bin/env bash
-
-__script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-
-# flow failures thru piped commands, disallow unrecognized variables, and exit on first failure
-set -euo pipefail
-
-source "$__script_dir/common.sh"
-
-__install_shared_runtime() {
-    local install_dir=$1
-    local version=$2
-    local channel=$3
-
-    local runtime_path="$install_dir/shared/Microsoft.NETCore.App/$version"
-    if [ ! -d "$runtime_path" ]; then
-
-        __verbose "Installing .NET Core runtime to $runtime_path"
-
-        "$__script_dir/dotnet-install.sh" \
-            --install-dir "$install_dir" \
-            --architecture x64 \
-            --shared-runtime \
-            --channel "$channel" \
-            --version "$version" \
-            $verbose_flag
-
-        return $?
-    else
-        echo -e "${GRAY}.NET Core runtime $version is already installed. Skipping installation.${RESET}"
-    fi
-}
-
-#
-# Main
-#
-
-verbose_flag=''
-if [ "$1" = "--verbose" ]; then
-    __is_verbose=true
-    verbose_flag='--verbose'
-    shift
-fi
-
-install_dir=$1
-
-if [ ! -z "${DOTNET_INSTALL_DIR:-}" ] && [ "${DOTNET_INSTALL_DIR:-}" != "$install_dir" ]; then
-    __verbose "install_dir = $install_dir"
-    __verbose "DOTNET_INSTALL_DIR = $DOTNET_INSTALL_DIR"
-    __warn 'The environment variable DOTNET_INSTALL_DIR is deprecated. The recommended alternative is DOTNET_HOME.'
-fi
-
-dotnet_in_path="$(which dotnet 2>/dev/null || true )"
-# The '-ef' condition tests if files are the same inode. This avoids showing the warning if users symlink dotnet into path
```

Comment:
This will be hard to do right in C#. It's only mildly important we do it, but it's one of the pieces of functionality we've lost in this port.